project/Dependencies.scala

```diff
@@ -11,7 +11,9 @@ object Dependencies {
     "io.cucumber"          % "cucumber-junit"    % "7.14.0"   % Test,
     "junit"                % "junit"             % "4.13.2"   % Test,
     "com.novocode"         % "junit-interface"   % "0.11"     % Test,
-    "uk.gov.hmrc"         %% "webdriver-factory" % "0.46.0"   % Test
+    "uk.gov.hmrc"         %% "ui-test-runner"    % "0.17.0"   % Test,
```

Comment:
ther eis a lot of cleanup here so I tried to see if I can do and I have this file now feel free to do more cleanup:
```scala
import sbt.*

object Dependencies {

  val test: Seq[ModuleID]    = Seq(
    "com.github.sbt" % "junit-interface" % "0.13.3",
    "io.cucumber"    % "cucumber-junit"  % "7.15.0",
    "io.cucumber"   %% "cucumber-scala"  % "8.20.0",
    "uk.gov.hmrc"   %% "ui-test-runner"  % "0.17.0"
  ).map(_ % Test)

  def apply(): Seq[ModuleID] = test

}


```

Reply:
thank you for cleaning up the app dep files should be updated now. Not sure if there is anything else to take out tho.